src/ast/function.rs

```diff
@@ -71,6 +71,30 @@ pub(crate) enum FunctionType<'a> {
     TextSearch(TextSearch<'a>),
     #[cfg(any(feature = "postgresql", feature = "mysql"))]
     TextSearchRelevance(TextSearchRelevance<'a>),
+    #[cfg(any(feature = "mysql"))]
+    UuidToBin,
+}
+
+/// Generates the function uuid_to_bin(uuid()) returning a binary uuid in MySQL
+/// ```rust
+/// # use quaint::{ast::*, visitor::{Visitor, Mysql}};
+/// # fn main() -> Result<(), quaint::error::Error> {
+
+/// let query = Select::default().value(uuid_to_bin());
+/// let (sql, _) = Mysql::build(query)?;
+///
+/// assert_eq!("SELECT uuid_to_bin(uuid())", sql);
+/// # Ok(())
+/// # }
+/// ```
+#[cfg(any(feature = "mysql"))]
+pub fn uuid_to_bin() -> Expression<'static> {
```

Comment:
I don't know quaint super well. But why does this have to be an expression over something like:
```
function!(UUIDToBin);
```

Reply:
Ah ok. That makes sense.

Reply:
That is only for expression structs, not enum members, I got super confussed by that as well